### PR TITLE
Create NodeJS.md

### PR DESCRIPTION
**Pull request by harshTTH**

### CODE REVIEW

**Review by saranshkataria — COMMENTED** [review]
Add a reference to this file in the README.md file as well

File: NodeJS.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# [NodeJS] Related Resources
```
Comment (saranshkataria) [inline NodeJS.md:1]: No need for the brackets

File: NodeJS.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+## For Beginners
+https://udemy.com/understand-nodejs/
```
Comment (saranshkataria) [inline NodeJS.md]: If the resource is a paid one, prepend it with a [PAID] prefix to let people know.

**Comment by harshTTH** [comment]
Added Reference to README.md file

**Review by saranshkataria — COMMENTED** [review]
@harshTTH please see inline comments in the file as well

**Comment by saranshkataria** [comment]
@harshTTH are you going to resolve the mentioned comments?

**Comment by harshitjuneja** [comment]
Resolved Conflicts. Merged to master.

**Pull request merged**